templates/server.template.yaml

```diff
@@ -89,52 +89,55 @@ Parameters:
   Fqdn:
     Description: The fully qualified domain name where the load balancer listens
     Type: String
+  BastionIngressCIDR:
+    Description: CIDR block Ingress for the Bastion Instance
+    Type: String
 Conditions:
   UsingDefaultBucket: !Equals
     - !Ref QSS3BucketName
     - aws-quickstart
   IsMarketplaceDeployment: !Equals
     - !Ref QSS3BucketName
     - !Join
-      - ""
+      - "-"
       - - "uipath-marketplace"
-        - !Ref "AWS::Region"
+        - "us-east-1"
 Mappings: 
   RegionMap: 
     ap-northeast-1:
-      x8664: "ami-0cd7548f41e94c7c3"
+      x8664: "ami-xxxxxxxxxxxx"
```

Comment:
There aren't valid AMI IDs. What's the purpose of this mapping if we don't have actual AMI Ids?

Reply:
We are using these templates for both marketplace and quickstart solutions. We discussed with the marketplace team and this was the result we came up with. 
These are only used in the marketplace deployment - check the if [here](https://github.com/aws-quickstart/quickstart-uipath-orchestrator/blob/47e5dd80ecfbb1fd8f651cfca33074a4e33faf45/templates/server.template.yaml#L666).

Reply:
I saw the IF Marketplace condition, however, I don't understand how the 'xxxxxx....' in the mapping is translating to the actual AMI Ids? May be I am missing something here. Pls email me with whom did you work with from MP team. I would like to better understand it.

Reply:
Any response to the above?

Reply:
We already sent the email 1 and a half months ago. I resent it now...